advanced_functionality/huggingface_deploy_instructpix2pix/deploy-instructpix2pix-to-sagemaker-ir.ipynb

```diff
@@ -0,0 +1,1281 @@
+{
```

Comment:
<p>Okay, just got word that I will need to host your model in our own bucket to avoid broken links. Just a question - do you have legal approval to use this model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3954/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I think, this will ruin the whole idea of this notebook. It shows customers how take the model from HF hub and not from their S3 bucket.

Reply:
The test on the endpoint uses is the same picture used in the example code under the model page https://huggingface.co/timbrooks/instruct-pix2pix. Inference recommender uses s3://sagemaker-sample-files/datasets/image/pets.

Do we need legal approval for those?

Reply:
<p>https://huggingface.co/timbrooks/instruct-pix2pix runs under MIT license. Do we still need extra approvals?</p>